src/main/java/cc/hyperium/addons/customcrosshair/gui/items/CCListDropBox.java

```diff
@@ -18,11 +18,9 @@
 package cc.hyperium.addons.customcrosshair.gui.items;
 
 import cc.hyperium.addons.customcrosshair.utils.CustomCrosshairGraphics;
-
-import java.awt.Color;
-
 import net.minecraft.client.gui.GuiScreen;
 
+import java.awt.*;
```

Comment:
It's our convention that we try to avoid wildcard imports, use explicit imports if possible.

Reply:
Plus there is no point in importing all of the `awt` package.  Only the classes specified.

Reply:
Yeah sorry about that, that's IntelliJ doing it automatically.